### ISSUE TITLE
bugs in the intro and first cutscene

### ISSUE THREAD

**Issue by staticz97** [CookiePLMonster/SilentPatchBully#22]
Happens in the game vanilla and with the silent patch, started after the April update of Windows 10.

https://www.youtube.com/watch?v=iBsJ6bxEoJs&feature=youtu.be

**Comment by PwnK** [comment]
I also have this bug. I have sounds working on the intro and the first cutscene but no video.

**Comment by dreamer1986** [comment]
Same problem, the intro video doesn't work.

**Comment by CookiePLMonster** [comment]
I quickly gave this a try and couldn't reproduce. For starters, MiniDumper shipped with a current release is kind of over-reactive (has been throwing all sorts of .dmp files during movies for some people), can you try again with the one from #4?

**Comment by staticz97** [comment]
Same issue with MiniDumper from #4 .

**Comment by Trium565** [comment]
I am also having this issue, I haven't tried it with the other cutscenes in the game but my guess is it would be the same.

**Comment by CookiePLMonster** [comment]
Build 2 is now out and Windows 10 received an update since the time of the last post - I assume this is still a problem, though?

For the record, I can't reproduce this (so far) on my Win10 rig.

**Comment by staticz97** [comment]
still having this problem , your windows 10 is in build 1803?

**Comment by CookiePLMonster** [comment]
Yep. If I recall correctly, movie player relies on `wmvcore.dll` - maybe you don't have it? If you are using Windows 10 in N or KN edition, that could be the case.

**Comment by staticz97** [comment]
I'm using windows 10 pro , I will try to test on another computer with the same build and see if the same problem happens.

**Comment by CookiePLMonster** [comment]
I also have a GTX 1070 here, so it's definitely not related to the GPU.

I'm pretty sure it's related to some specific combination of software or system settings -- but not yet sure what.

**Comment by Trium565** [comment]
Yea Im sure it isnt a hardware issue. I have an i7-8700k, GTX 1080Ti and 32Gb of RAM. It must be something to do with the system settings but I cant figue out what. Ill take another look later on to see if I can find anything.

**Comment by CookiePLMonster** [comment]
DPI scaling maybe? Some overlay software, like MSI Afterburner?

**Comment by staticz97** [comment]
I already tested with the pc newly formatted only drivers and steam installed and it happened the same thing (silent patch build 1), only difference of my notebook is that it is windows home not the pro .

**Comment by CookiePLMonster** [comment]
Test with Build 2 please - it has some new fixes and a better MiniDumper.

**Comment by staticz97** [comment]
I formatted the pc, I just downloaded the drivers through windows update (installed an old nvidia driver from 2017)  game worked perfectly ,
then installed the latest drivers from nvidia, , and the bug came back , so I'm pretty sure the problem is the current drivers for nvidia.

**Comment by CookiePLMonster** [comment]
I'm on 397.64 at the moment, is this old/new enough?

**Comment by staticz97** [comment]
It's kind of new, about 2 months ago.

**Comment by Trium565** [comment]
Yea I am also using both MSI Afterburner and the latest Nvidia Drivers. Although I doubt MSI Afterburner would be causing any problems. It must be the drivers, although I havent managed to test it on other versions as of yet.

**Comment by Trium565** [comment]
Ok so I have just had a look and I have Windows 10 Home Edition (2018), the latest Nvidia Drivers and The latest release of this patch. It is still happening, even after not using MSI Afterburner, but I dont want to have to downgrade my drivers because of the new patches with the latest games that I use. I am honestly so stuck right now. I have even tried running it in Compatibility mode but to no avail.

**Comment by PwnK** [comment]
@Trium565 This bug only affects intro and the first cutscene all other cutscenes after that are fine so why don't you just ignore this bug and play the game? I've played about half of game with this bug and all other cutscenes are fine.

**Comment by Trium565** [comment]
@PwnK Thanks for letting me know. I didn't know whether it just affected the first cutscene or every cutscene as I didnt test it(I didnt want to F*** up my game lol). But thanks dude, I hope they fix it soon though. It is weird I gotta admit.

**Comment by CookiePLMonster** [comment]
I'm really unsure on what causes that - I haven't upgraded my drivers to the newest yet, but 397.xx should be new enough for this bug to also surface **if** it was caused by that.

Then again, we are talking about a movie player - they are notoriously bad in older games, and Bully is one of them. GTA III/VC/SA are even worse when it comes to that.

My advice would be to just use `--skipMoviesASAP` commandline option until this is resolved - you'll save on startup times and also won't have to look at broken movies :)

**Comment by Silanda** [comment]
I just thought I'd comment on this because I found a solution on my system (Windows 10, GTX 1080). This is not the only game I've had this problem with. I don't know the cause, but all I know is that it didn't used to happen, and the videos very rarely play without any fix. 

The other game I had this problem with was Onechanbara Z2: Chaos, and I found that if I turned 4x DSR on in the Nvidia control panel and then set my desktop resolution to 3840x2160 (i.e. the 4x DSR res on my system) the videos would be displayed. I don't need to do that for Bully, all I need to do is select 3840x2160 as the resolution in game and everything is fine. Onechanbara is another DirectX 9 game if that could mean anything.

Here's a curious thing though: it only works if I use DSR. If I turn DSR off and create a identical custom resolution, at the same refresh rate, and select that, the videos are still MIA.

**Comment by CookiePLMonster** [comment]
Interesting... I don't know why enabling DSR would change anything here. When using 1080p (and DSR off) they do not show up?

**Comment by Silanda** [comment]
No they don't, but it gets a bit stranger. They don't work properly with the Steam version, but since I last posted I got hold of a physical disc copy. The videos in the 1.2 patched disc version work just fine at any resolution, so the problem on my system only affects the Steam version. 

All Steam overlays are turned off too, so I'm a little puzzled.

**Comment by poohlya** [comment]
If I use this parameter "SkipIntroSplashes=1" in GTA: SA Silent Patch ini videos show up... But with Bully only helps ALT+TAB (sometimes)...

**Comment by Shadic93nh** [comment]
I tried DSR but no luck on my end. It only opened movies after first fresh install then it messes up. I hope one day we will get cutscenes to play normally
